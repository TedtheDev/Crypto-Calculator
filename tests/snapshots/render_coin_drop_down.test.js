import React from 'react';
import CoinDropDown from '../../src/components/coin_drop_down';
import renderer from 'react-test-renderer';

test('Coin Drop Down renders correctly', () => {
    const component = renderer.create(
        <CoinDropDown coins={[{name: 'Coin 1', id: 'coin1'},{name: 'Coin 2', id: 'coin2'}]}/>
    );

    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
});