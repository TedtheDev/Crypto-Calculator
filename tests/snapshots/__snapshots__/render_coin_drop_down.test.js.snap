// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Coin Drop Down renders correctly 1`] = `
<div>
  <div
    aria-busy={undefined}
    aria-disabled={undefined}
    aria-expanded={false}
    className="ui fluid search selection dropdown"
    onBlur={[Function]}
    onChange={[Function]}
    onClick={[Function]}
    onFocus={[Function]}
    onMouseDown={[Function]}
    role="combobox"
    tabIndex={undefined}
  >
    <input
      aria-autocomplete="list"
      autoComplete="off"
      className="search"
      onChange={[Function]}
      style={
        Object {
          "width": undefined,
        }
      }
      tabIndex={0}
      type="text"
      value=""
    />
    <div
      aria-live="polite"
      className="default text"
      role="alert"
    >
      Select Crypto
    </div>
    <i
      aria-hidden="true"
      className="dropdown icon"
      onClick={[Function]}
    />
    <div
      aria-multiselectable={undefined}
      className="menu transition"
      role="listbox"
    >
      <div
        aria-checked={false}
        aria-disabled={undefined}
        aria-selected={true}
        className="selected item"
        onClick={[Function]}
        role="option"
        style={
          Object {
            "pointerEvents": "all",
          }
        }
      >
        <span
          className="text"
        >
          Coin 1
        </span>
      </div>
      <div
        aria-checked={false}
        aria-disabled={undefined}
        aria-selected={false}
        className="item"
        onClick={[Function]}
        role="option"
        style={
          Object {
            "pointerEvents": "all",
          }
        }
      >
        <span
          className="text"
        >
          Coin 2
        </span>
      </div>
    </div>
  </div>
</div>
`;
